mopidy_spotify/web.py

```diff
@@ -216,15 +216,19 @@ def _prepare_url(self, url, *args, **kwargs):
             scheme, netloc = b.scheme, b.netloc
             path = os.path.normpath(os.path.join(b.path, u.path))
             query = urllib.parse.parse_qsl(b.query, keep_blank_values=True)
-            query.extend(urllib.parse.parse_qsl(u.query, keep_blank_values=True))
+            query.extend(
+                urllib.parse.parse_qsl(u.query, keep_blank_values=True)
+            )
 
         for key, value in kwargs.items():
-            if isinstance(value, unicode):
+            if isinstance(value, str):
                 value = value.encode("utf-8")
```

Comment:
"utf-8" is the default encoding in Python 3, so all `.decode()` and `.encode()` calls can be changed to not have any arguments.

Reply:
I got rid of this entirely since Python3 `urlencode()` is happy with unicode.